Casks/l/logitech-camera-settings.rb

```diff
@@ -10,6 +10,14 @@
   livecheck do
     url "https://support.logi.com/api/v2/help_center/en-us/articles.json?label_names=webcontent=productdownload,websoftware=9bf6fc93-8e0b-11e9-a62b-cb4c7fb3c2e2"
     regex(%r{/LogiCameraSettings[._-]?(\d+(?:\.\d+)+)\.pkg}i)
```

Comment:
```suggestion
    regex(/href=.*?LogiCameraSettings[._-]v?(\d+(?:\.\d+)+)\.pkg/i)
```

`logitech-presentation` uses a typical regex pattern and we can do the same thing for `logitech-camera-settings`, so we may as well bring it in line.